cdap-data-fabric/src/main/java/co/cask/cdap/data2/admin/DefaultAdmin.java

```diff
@@ -14,7 +14,7 @@
  * the License.
  */
 
-package co.cask.cdap.internal.app.runtime;
+package co.cask.cdap.data2.admin;
```

Comment:
I wanted to move it to ``cdap-common``. However:
* It was earlier in app fabric. It has a dependency on ``DatasetFramework``. 
* I had to move ``AuthorizationModule`` from ``cdap-app-fabric`` to ``cdap-data-fabric``.
* The ``AuthorizationModule`` cannot be in ``cdap-security`` because it uses ``DynamicDatasetCache``, ``SystemDatasetInstantiator``, ``DefaultAdmin``, etc to construct ``AuthorizationContext``.